packages/@react-aria/test-utils/package.json

```diff
@@ -22,6 +22,7 @@
     "url": "https://github.com/adobe/react-spectrum"
   },
   "dependencies": {
+    "@react-aria/utils": "^3.28.1",
```

Comment:
I don't think we should have a dependency going in this direction, test-utils should never depend on our actual packages
too easy to make a circular dependency